NotificationSystem/Services/EmailSender.cs

```diff
@@ -43,21 +44,19 @@ public EmailSender(ISendGridClient sendGridClient, string emailFrom, bool sandbo
 
         public void ParsePayload(string jsonBody)
         {
+            notification = JsonConvert.DeserializeObject<EmailNotification>(jsonBody);
             Notification = JsonConvert.DeserializeObject<EmailNotification>(jsonBody);
```

Comment:
why use 2 similar variables?

Reply:
Also the capitalized one is not used

Reply:
Basically the Notification was used to check the value while testing, but the way it is coded now is indeed not the right way. We've encapsulated the field now with a getter.